### ISSUE TITLE
Add guidance for the correct capitalization and short form for referring to basic authentication

### ISSUE THREAD

**Issue by michelle-purcell** [redhat-documentation/supplementary-style-guide#206]
### Problem
In Red Hat customer-facing documentation the term "basic authentication" is capitalized inconsistently as outlined in the examples below. Guidance is needed for when to use or not use each variation.

Lowercase **_basic authentication_** is the most commonly used across Red Hat docs, especially when referring to the authentication method in general. All lowercase is also preferred by the CCS style council in this gChat discussion: https://chat.google.com/room/AAAAAjCmTJI/AVRNWBJ3bwI

However, SMEs for Quarkus have pushed back on the lowercase form and prefer **_Basic authentication_**, when specifically referring to authentication that is based on the ['Basic' HTTP Authentication Scheme' RFC specification](https://www.rfc-editor.org/rfc/rfc7617.txt), which uses **_Basic authentication_** throughout the definition. 
![image](https://user-images.githubusercontent.com/92924207/185329030-8f9e554c-2a4e-4080-a9d6-93bc1f20b81d.png)

**BA** is aso a commonly used short form for the term.

### Request
Provide guidance in the style guide for the correct capitalization and also guidance for using the initialism.

### Examples of inconsistency:

**_basic authentication_**

![image](https://user-images.githubusercontent.com/92924207/185319191-e5ed5f54-8414-4f03-acaa-9078ca06d77c.png)
Source: https://access.redhat.com/documentation/en-us/red_hat_insights/2022/html/configuring_basic_authentication_for_red_hat_insights/con-insights-client-config-basic-auth-introduction_insights-client-config#con-insights-client-config-basic-auth-when-to_insights-client-config

**_Basic authentication_**

![image](https://user-images.githubusercontent.com/92924207/185317848-ef2e83ff-9894-4e4f-8be6-2053f548e3e3.png)
Source: https://access.redhat.com/documentation/en-us/cost_management_service/2022/html/adding_an_openshift_container_platform_source_to_cost_management/assembly-adding-openshift-container-platform-source#installation-overview_adding-an-ocp-source

**_Basic Authentication_**

![image](https://user-images.githubusercontent.com/92924207/185331512-ce5f448b-8979-423e-88bb-199d500b3ee0.png)
Source: https://www.ibm.com/docs/en/sva/7.0.0?topic=processing-configuring-basic-authentication

![image](https://user-images.githubusercontent.com/92924207/185318830-ed641cb8-d90e-4935-b342-6136865433b8.png)
Source: https://access.redhat.com/articles/4038251

**Shortform: **_BA_****
![image](https://user-images.githubusercontent.com/92924207/185333516-d6616c18-7707-4892-8f4e-7e40468c8cbc.png)
Source: https://www.ibm.com/docs/en/sva/7.0.0?topic=processing-configuring-basic-authentication

**_BASIC authentication_** 

![image](https://user-images.githubusercontent.com/92924207/185318550-1c7cf993-a38f-42ea-9826-c0a513dbd0a4.png)
Source: https://access.redhat.com/documentation/en-us/red_hat_jboss_fuse/6.0/html/security_guide/cameljetty-basicauth

**Comment by bburt-rh** [comment]
@michelle-purcell I agree that we need to add guidance for this so that we're consistent across all RH docs. What are you proposing as the standard? 

We could use "basic authentication" and justify it for a couple of reasons
- This is the most common use I've seen in RH docs already (thus, path of least resistancd).
- The W3C and original HTTP 1.0 spec use it in lower case on these pages:
- - https://www.w3.org/MarkUp/Forms/wiki/HTTP_Authentication
- - https://www.w3.org/Protocols/HTTP/1.0/spec.html#BasicAA
- - https://www.rfc-editor.org/rfc/rfc1945#section-11.1
On the other hand, the IETF standard for BA capitalizes "Basic", and this is the more recent official standards doc:
- https://www.rfc-editor.org/rfc/rfc7617
My own thought is that it should be "Basic authentication" because that's what's used in the latest IETF RFC. Capitalizing it makes it sound like something official rather than something generic (i.e. no different than saying "simple authentication").

@redhat-documentation/ccs-style-council: WDYT?

**Comment by bergerhoffer** [comment]
I think that this one is a tough one. I originally was on the side of lowercasing "basic" because that's what we do in OpenShift. We also do have a "request header" authentication which is lowercase. The rest are uppercase (Keystone, GitHub, LDAP, etc.).

I agree that the hard part here is that "basic" sounds like it isn't something specific, but "request header" does sound like something specific.

So now, due to the confusion that lowercase "basic" could create, I am in favor of recommending capitalized "Basic authentication".

**Comment by bburt-rh** [comment]
@bergerhoffer Because we're in agreement, would the next steps be to post to the gchat channel and solicit further input? I'd like to know what others think before we move forward with developing official guidance. 

Also, whatever the guidance is that we do define, does CCS have a policy or practice regarding updating existing docs that use a term in a different way than what the SSG newly specifies?

**Comment by bergerhoffer** [comment]
@bburt-rh there was actually a discussion in the CCS style council chat around this. I say that it would be a good idea to either revive that thread with this new direction, or start a new thread to see if others are on board.

As for if we have a policy on updating existing docs, the answer is no. We make the guidance, and changes to guidance as necessary, and then communicate it out to CCS. It's then up to each writing team to decide if and when to make the switch.

**Comment by michelle-purcell** [comment]
@bburt-rh @bergerhoffer Thank you for looking into this and giving it airtime :100: 

@bburt-rh To answer the above q: _What are you proposing as the standard?_

Our team consensus leans more towards **Basic authentication**. If the content is specifically talking about the technical aspects of the RFC for HTTP Basic authentication, then Basic authentication seems to be correct.

However, the lowercase form of **basic authentication** is not incorrect either because it is understood by most readers to mean **HTTP Basic authentication**. I would also agree that it is also correct when used in a more general sense to describe simpler authentication methods.

This term comes up a good bit in our peer and SME reviews so having guidance on when to use the above would reduce the review churn and provide consistency across Red Hat docs. We would also like the stye guide to clarify the forms that are incorrect, for example, **BASIC authentication** and **Basic Authentication**, again to reduce the churn.

Many thanks

**Comment by bredamc** [comment]
Maybe use **HTTP Basic authentication** on first occurrence, and then **Basic authentication** from then on?

Definitely not "BASIC", it's not related to the computer language.

**Comment by bergerhoffer** [comment]
Discussed this at the 30 November SSG meeting. No objections go going with "Basic authentication" capitalization.

As for expanding on first occurrence, I think the right order would be "Basic HTTP authentication" per the spec: https://www.rfc-editor.org/rfc/rfc7617. 

@bburt-rh has volunteered to open the PR for this (thank you!)
